fxa-auth-db-server/lib/bufferize.js

```diff
@@ -15,21 +13,24 @@ function unbuffer(object) {
   return object
 }
 
-function bufferize(object, ignore) {
+function bufferize(object, onlyTheseKeys) {
   var keys = Object.keys(object)
+  if (onlyTheseKeys) {
+    keys = keys.filter(function(key) {
+      return key in onlyTheseKeys
+    })
+  }
   for (var i = 0; i < keys.length; i++) {
     var key = keys[i]
     var value = object[key]
-    if (ignore.indexOf(key) === -1 && typeof value === 'string' && HEX_STRING.test(value)) {
-      object[key] = Buffer(value, 'hex')
-    }
+    object[key] = Buffer(value, 'hex')
```

Comment:
I reckon we should add `new` to `Buffer` instantiations like this, whenever we encounter them. Because, [as of node 7, `Buffer`-without-`new` is deprecated](https://nodejs.org/en/blog/release/v7.0.0/#notable-changes).